roles/openshift_logging/tasks/generate_configmaps.yaml

```diff
@@ -8,6 +8,11 @@
         src: elasticsearch.yml
         dest: "{{mktemp.stdout}}/elasticsearch.yml"
 
+    - lineinfile: 
```

Comment:
i still think we should only be running this when es_ops_allow_cluster_reader is true. save us a cycle when it's 'false'